### DISCUSSION TITLE
Uploaded document is not scanned.

### DISCUSSION THREAD

**Discussion by UUSR** [open-webui/open-webui#2969] [General]
# Bug Report

## Description

**Bug Summary:**
- When documents are uploaded, they go into the .\backend\uploads folder, but they do not appear in the .\backend\data\docs folder until they are forced to be added, only after which they are scanned in the program.

**Steps to Reproduce:**
- Simply add any document to the workspace in any way, either through chat or through the documents workspace.

**Actual Behavior:**
- The uploaded document is not scanned and does not go to .\backend\data\docs

## Environment

- **Open WebUI Version:** v0.2.1
- **Ollama (if applicable):** 0.1.42

- **Operating System:** Windows 11
- **Browser (if applicable):** Chrome 125.0.6422.142

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Screenshots

**Screenshots (if applicable):**
![2024-06-10_051302](https://github.com/open-webui/open-webui/assets/17599033/172edfd3-505e-4c55-81c4-9959673d56e9)

## Installation Method
Method you used to install the project manual installation.

**Comment by Hugobox** [comment]
Not sure if related, but I have to upload my documents twice for them to appear in the UI

**Comment by garyman123** [comment]
Same problem.Did you find how to solution?https://github.com/open-webui/open-webui/discussions/3633#discussion-6898561

**Reply by UUSR** [reply]
Update to the latest version, everything already works.
